raincloudy/faucet.py

```diff
@@ -237,9 +233,10 @@ def watering_time(self, value):
 
     @property
     def droplet(self):
-        """Return droplet URL from zone"""
-        value = 'droplet_zone_{}'.format(self.id - 1)
-        return "{0}{1}".format(API_URL, self._lookup_attr(value))
+        return None
+        # """Return droplet URL from zone"""
+        # value = 'droplet_zone_{}'.format(self.id - 1)
+        # return "{0}{1}".format(API_URL, self._lookup_attr(value))
```

Comment:
This doesn't exist on the json API anymore. We'd need to scrape it from the `/home` dump. Happy to do so but non-trivial compared to the other work in this PR.